app/src/main/java/org/ometa/lovemonster/service/LoveMonsterClient.java

```diff
@@ -340,6 +346,31 @@ void onAuthenticationFailure() {
         });
     }
 
+    public void getUserFromUsername(@NonNull String username, @NonNull final UserLookupResponseHandler responseHandler) {
+        final URIBuilder url = buildUrl("/api/v1/users/" + username);
```

Comment:
What's the proper way to build this URL in Java?  Feels like a user could search for "../foobar" and screw things up.